lib/salus/scanners/gitleaks.rb

```diff
@@ -0,0 +1,109 @@
+require "salus/scanners/base"
+require "json"
+
+# Gitleaks scanner integration. Detects hardcoded secrets like passwords,
+# api keys, and tokens in git repos.
+# https://github.com/zricethezav/gitleaks
+
+module Salus::Scanners
+  class Gitleaks < Base
+    def run
+      Dir.chdir(@repository.path_to_repo) do
+        copts = config_options
+        shell_return = run_shell(
+          'gitleaks '\
+          '--path . '\
+          '--format sarif '\
+          '--report /dev/stdout '\
```

Comment:
Is `--report` a required config option?

Reply:
`--report` is required to output to `stdout`. Otherwise, only debug data is output (to `stderr`).